refinery/ui/source/js/data-set-about/partials/details.html

```diff
@@ -41,16 +50,18 @@
         type="submit"
         class="btn btn-xs btn-primary"
         ng-click="ADCtrl.updateDataSet(
-        'accession', ADCtrl.updatedField.accession)"
-        ng-disabled="!dataSetAccessionForm.$valid">
+        'accession', ADCtrl.editedDataSet.accession)"
+        ng-disabled="!dataSetAccessionForm.$valid ||
+        ADCtrl.editedDataSet.accession == ADCtrl.dataSet.accession">
```

Comment:
Shouldn't make a difference here, but should `===` be the default?

Reply:
@mccalluc https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Typically for HTML comparisons, I use the abstract == because I want the same value. In forms, if I want validation I will add them for types (ie booleans or integers). In JS code, yes.
Thoughts?